contracts/vendor/SafeCall.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+/**
+ * @title SafeCall
+ * @notice Perform low level safe calls. See https://github.com/nomad-xyz/ExcessivelySafeCall
+ */
+library SafeCall {
+    /**
+     * @notice Perform a low level call without copying any returndata
+     *
+     * @param _target   Address to call
+     * @param _value    Amount of value to pass to the call
+     * @param _calldata Calldata to pass to the call
+     */
+    function call(address _target, uint256 _value, bytes memory _calldata) internal returns (bool) {
+        bool _success;
+        uint256 _gas = gasleft();
+        assembly {
+            _success := call(
+                _gas, // gas
+                _target, // recipient
```

Comment:
Why not rename `_target` to `_recipient` and avoid the comment?

Reply:
It's pretty much the same thing, the comment probably isn't even needed